domain/src/main/assets/welcome.json

```diff
@@ -11,7 +11,24 @@
   "states": {
     "END": {
       "classifier_model_id": null,
+      "recorded_voiceovers": {
+        "voiceovers_mapping": {
+          "content": {
+            "hi-en": {
+              "file_size_bytes": 0,
```

Comment:
Here & below: I suggest setting the sizes for when we might use them in the future.